cypress/e2e/types/constants.ts

```diff
@@ -69,7 +69,7 @@ export const stakeholders = "Stakeholders";
 export const stakeholderGroups = "Stakeholder groups";
 export const subversion = "Subversion";
 export const tag = "Tag";
-export const tagCount = "Tags";
+export const tagCount = "Tag count";
```

Comment:
This breaks the sorting Test case in applications.
We need to add another const if we need "Tag count" as well .

Reply:
The application sort test used const tags and it did fixed the test. 
![Screenshot from 2023-11-01 09-26-28](https://github.com/konveyor/tackle-ui-tests/assets/87791967/29c6599c-ba69-49d7-9321-831dce3ce76d)

Reply:
Also, did ran the other affected test tagCategory sort and it Passed